assets/hip-1056/protobuf/stream/trace/smart_contract_service.proto

```diff
@@ -0,0 +1,210 @@
+/**
+ * # Smart Contract Service
```

Comment:
I don't think the file name implied the contents, it's not a service definition.

Reply:
Switchd to `Smart Contract Service Trace Data`